sensor_msgs/Image image
---
corobot_follower/OpenPoseHumanList human_list